src/windows/wslaservice/exe/WSLASession.cpp

```diff
@@ -19,6 +19,36 @@ Module Name:
 #include "ServiceProcessLauncher.h"
 #include "WslCoreFilesystem.h"
 
+namespace
+{
+    constexpr const char* nerdctlPath = "/usr/bin/nerdctl";
+
+    HRESULT TransferFileContent(HANDLE from, HANDLE to)
```

Comment:
FYI we already have a helper to do this: https://github.com/microsoft/WSL/blob/25572cc168b5a54e18a63104d5cebc478fd6677d/src/windows/common/relay.hpp#L39 

Also whenever we call it we should make sure to pass the VM termination event ass `ExitHandle` so the relay is interrupted when vm shuts down